conf/broker.conf

```diff
@@ -450,6 +450,10 @@ brokerServiceCompactionMonitorIntervalInSeconds=60
 # Using a value of 0, is disabling compression check.
 brokerServiceCompactionThresholdInBytes=0
 
+# Timeout for the compaction phase one loop.
+# If the execution time of the compaction phase one loop exceeds this time, the compaction will not proceed.
+brokerServiceCompactionPhaseOneLoopTimeInSeconds=10
```

Comment:
Most of the other timeouts are set to 30seconds. Maybe we could also switch this default value to 30.

Reply:
Make sense.